code/game/objects/items/stacks/cash.dm

```diff
@@ -2,7 +2,7 @@
 	name = "space cash"
 	singular_name = "bill"
 	icon = 'icons/obj/economy.dmi'
-	icon_state = "spacecash"
+	icon_state = null
```

Comment:
(because players are more likely to report an incorrectly spawned base path item when it's a missing texture and has a blinking FUCK than when it just looks like a 1 credit bill)